src/MinimumNodeService.cpp

```diff
@@ -82,6 +82,8 @@ void MinimumNodeService::initSetupFromNormal()
   // DEBUG_SERIAL << F("> reverting to Uninitialised mode") << endl;
   requestingNewNN = true;
   controller->sendMessageWithNN(OPC_NNREL);
+  controller->sendGRSP(OPC_MODE, getServiceID(), GRSP_OK);
```

Comment:
One side effect of this change is that you get a GRPS(OK) also when you press the push button. Not sure this is what you want.

Reply:
Hadn't thought of that! Now sorted

Reply:
That is why we do code reviews. :)

Reply:
Yes. And I'm pleased to be a part of something that is working well!